scipy/stats/_continuous_distns.py

```diff
@@ -9244,6 +9244,35 @@ def _entropy(self, kappa):
         return (-kappa * sc.i1e(kappa) / sc.i0e(kappa) +
                 np.log(2 * np.pi * sc.i0e(kappa)) + kappa)
 
+    @extend_notes_in_docstring(rv_continuous, notes="""\
+        The limits of integration of the vonmises distribution
+        default to the 2pi interval centered at loc
+        (e.g. [-pi,pi] when ``loc=0``).
+
+        >>> r = vonmises(kappa=1).expect(lambda x: np.exp(1j*x))
+        >>> np.angle(r)
+        0.0
+
+        The ``vonmises::expect`` method, as shown above, uses the
+        ``complex_func`` feature of ``integrate.quad`` to evaluate
+        integral.\n\n""")
```

Comment:
There are a few little issues here (e.g. technically I think the limits of integration are numbers, not an interval; this means that the integral is taken over an interval), so I'd suggest
```suggestion
        The default limits of integration are endpoints of the interval
        of width ``2*pi`` centered at `loc` (e.g. ``[-pi, pi]`` when
        ``loc=0``). The integrand is always treated as a complex-valued
        function, so the dtype of the return value will always be complex.

        >>> res = vonmises(loc=2, kappa=1).expect(lambda x: np.exp(1j*x))
        (-0.18576377217422957+0.40590124735052263j)

        >>> np.angle(res)
        2.0
        \n\n""")
```
Let me know if you'd like more justification.